src/cosmology/compat/camb/__init__.py

```diff
@@ -56,7 +56,15 @@ def hubble_distance(self) -> Array:
 
     def H(self, z: Array | float) -> Array:
         """Hubble parameter at redshift *z*."""
-        return np.array(self.results.hubble_parameter(z))
+        return np.array(
```

Comment:
So this H doesn't have neutrinos? or light? I'm not seeing any $a^{-4}$ nor anything transitioning into matter.